images/iscsi-demo-target-tgtd/Dockerfile

```diff
@@ -21,22 +21,15 @@ FROM fedora:27
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 ENV container docker
 
-RUN dnf -y update && dnf -y install scsi-target-utils bzip2 qemu-img \
-    && dnf -y clean all
+RUN dnf -y install scsi-target-utils bzip2 e2fsprogs
 
-RUN mkdir -p /volume
+RUN mkdir -p /images
 
 # Add alpine image
-RUN curl \
-      http://dl-cdn.alpinelinux.org/alpine/v3.7/releases/x86_64/alpine-virt-3.7.0-x86_64.iso \
-      > /volume/alpine.iso
+ADD http://dl-cdn.alpinelinux.org/alpine/v3.7/releases/x86_64/alpine-virt-3.7.0-x86_64.iso /images/1-alpine.img
 
 # Add cirros
-RUN curl \
-      https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img \
-      > /volume/cirros.img \
-      && qemu-img convert -O raw /volume/cirros.img /volume/cirros.rawa \
-      && rm volume/cirros.img
+ADD https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img /images/2-cirros.img
```

Comment:
I'm in favor of getting rid of cirros in both of these demos in order to reduce size. Alpine should be enough to prove this works now.

Reply:
@davidvossel Will check where we are using it